src/main/java/com/google/api/codegen/viewmodel/OutputView.java

```diff
@@ -20,28 +20,28 @@
 public interface OutputView {
 
   public enum Kind {
-    ASSIGNMENT,
     COMMENT,
+    DEFINITION,
```

Comment:
WDYT about the idea above of making these all verbs that correspond to the input spec?